src/main/java/de/tum/in/www1/exerciseapp/web/rest/ParticipationResource.java

```diff
@@ -144,7 +144,7 @@ public ParticipationResource(ParticipationService participationService, Particip
      * @throws URISyntaxException if the Location URI syntax is incorrect
      */
     @PutMapping("/participations")
-    @PreAuthorize("hasAnyRole('TA', 'ADMIN')")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
     @Timed
     public ResponseEntity<Participation> updateParticipation(@RequestBody Participation participation) throws URISyntaxException {
```

Comment:
This should only be possible for TAs and instrcutors of the corresponding exercise (and admins)

Reply:
Same here. I believe this one was also never touched after having been generated.

Reply:
Done,